multi-patchs.tex

```diff
@@ -246,11 +385,10 @@ \section{Advection equation}
 \end{center}
 
 \paragraph{Further questions on outside feet.}
-Problem: The mapping from the logical to the physical domain can be costly to invert. 
+Problem: the mapping from the logical to the physical domain is not always easily invertible. 
 
 \begin{itemize}
-	\item For an advection in the physical domain, if the feet is outside of the domain, how to get the feet in the logical domain of another patch 
-			(inverting the mapping for the other patch might be costly)?
+	\item For an advection in the physical domain, if the feet is outside of the domain, how to get the feet in the logical domain of another patch (could be a not invertible mapping for the other patch)?
```

Comment:
No, the mappings we consider are always invertible almost everywhere with the only exception of O-points. The problem is that the computation is computationally expensive. Your characteristic foot will not land at the O-point when it leaves a patch, I am pretty sure

Reply:
I think I didnt  pull your changes. I didnt wanted to change this sentence.